.github/CONTRIBUTING.md

```diff
@@ -252,7 +252,7 @@ describe('Segment', () => {
   common.propValueOnlyToClassName(Segment, 'color')
   common.propKeyOnlyToClassName(Segment, 'basic')
   common.propKeyAndValueToClassName(Segment, 'floated')
-  common.propKeyOrValueToClassName(Segment, 'padded')
+  common.propKeyOrValueAndKeyToClassName(Segment, 'padded')
```

Comment:
Updated this test name to be consistent with `useKeyOrValueAndKey()`, fixed #481.